docs/pages/migrating.mdx

```diff
@@ -0,0 +1,168 @@
+
+- Roles
+- Local users
+- SSO authentication connectors
+
+Review the [dynamic resources](https://goteleport.com/docs/reference/resources/) list to see if any other resources need to be migrated.  Some common dynamic resources includes:
+- windows_desktop
+- apps
+- dbs
+- login_rule
+
+</Details>
+
+<Admonition
+  type="tip"
+  title="Note"
+>
+
+  Trusted Clusters are no longer supported in Teleport Cloud.  You will not be able to migrate trusted cluster resources.
+
+</Admonition>
+
+
+To achieve this:
+- Log in to your existing self-hosted Teleport cluster and export a collection of the above-mentioned dynamic resource configuration using the tctl administrative tool.  An example is shown below:
+
+```code
+$ tsh login --proxy=teleport.example.com --user=admin@example.com
+$ tctl get roles > roles.yaml 
+$ tctl get users > users.yaml 
+```
+
+- Once you have the resource configuration file from the above, proceed to log in to your Teleport Cloud tenant with an admin user and create the resources from the exported files
+
+```code
+$ tsh login --proxy=example.teleport.sh --user=admin@example.com
+$ tctl create -f  roles.yaml
+$ tctl create -f  user.yaml
+```
+
+For your SSO auth connector, most SSO integrations only work for a single configured endpoint. It is recommended to create a separate SSO connector in your Identity Provider specifically for the Teleport Cloud endpoint, and configure a new Auth Connector in the Teleport Cloud tenant.
+
+
+### Migrate Teleport Agents.
+
+To ensure a successful migration process across your infrastructure, start by cataloging the various services you're managing with Teleport, The following resources should be considered for migration:
+Teleport Agents (SSH, Kubernetes, Databases, Applications, Desktops)
+Machine ID Bots
+Access Request Plugins
+
+The following procedure is required to successfully migrate your Teleport agents:
+
+- Log in to your Teleport Enterpise cloud Tenant, You'll need to authenticate to access your Teleport Cloud resources and management interfaces.
+
+- Obtain a valid join token: We recommend using a [Delegated Join Method](https://goteleport.com/docs/reference/join-methods/#delegated-join-methods)
+
+If using ephemeral tokens, ensure you specify the appropriate token type to match the Teleport services. Token types can include `node`, `app`, `Kube`, `DB`, `WindowsDesktop`, and others depending on the service you wish to join to your Teleport cluster.
+
+In the following example, a new token is created with the TTL of fifteen minutes:
+
+```code
+$ tctl tokens add --type node,app,db --ttl 15m
+```
+
+In this command, we assigned the token the `node`, `app` and `db` type, indicating that it will belong to an SSH Service, Database Service and Application Service instance.

```

Comment:
```suggestion
In this command, we assigned the token the `node`, `app` and `db` type, indicating that it will allow an agent to join which is running the Teleport `ssh_service`, `db_service` and `app_service`.

```